### ISSUE TITLE
strongly recommend to use Tensorflow or Pytorch as computing platform in the future　:) #3

### ISSUE THREAD

**Issue by lemonadeww** [hnanhtuan/selectiveConvFeature#2]
Hi hnanhtuan, your code is written very elegant and with amazing performance. Since matconvnet is not friendly to quickly use and development community is not strong, and its training efficiency and deployment would be inferior to the above platforms. Python is more convenient to process network and data. Would you be so kind to provide codes of Tensorflow or Pytorch version?

Thanks for your code sharing again !

**Comment by hnanhtuan** [comment]
Thanks for your interest in my work and your comment. 
In fact, the feature extraction part, you can use any deep learning framework to extract image Convolution feature maps, and then save each feature map as .mat file with the corresponding name (image file name).
The main file is based on the feature map .mat file. There is nothing related to the matconvnet.

**Comment by lemonadeww** [comment]
Thank you for your reply.
I also have a question that MAX-mask is done on the feature map from pool5 of VGG16, but it seems that the mask has the same size as the original image from Fig.1 (d)-(e) in your paper 'Selective Deep Convolutional Features for Image Retrieval'?

**Comment by lemonadeww** [comment]
I have fixed it. Sorry to bother you.
